gateway/src/apicast/cli/environment.lua

```diff
@@ -40,6 +41,16 @@ local function parse_nameservers()
     end
 end
 
+local function cpus()
+    local cores = tonumber(resty_env.get('NUMBER_OF_CORES'))
```

Comment:
Do we need this? Is there any reason someone would use `NUMBER_OF_CORES` instead of the existing `APICAST_WORKERS`?

Reply:
Also, if we start accepting this ENV, we should document it https://github.com/3scale/apicast/blob/master/doc/parameters.md

Just realized that `APICAST_WORKERS` is not there.

Reply:
All software collection images use NUMBER_OF_CORES, so I thought it would make sense to follow that. 

But probably it does not make sense because the SCL images override `NUMBER_OF_CORES` with own value.

Will remove it.